sacrebleu/tokenizers/tokenizer_spm.py

```diff
@@ -11,11 +11,29 @@
 sacrelogger = logging.getLogger('sacrebleu')
 
 
+SPM_MODELS = {
+    "spm": {
+        "url": "https://dl.fbaipublicfiles.com/fairseq/models/flores/sacrebleu_tokenizer_spm.model",
+        "signature": "spm-flores",
+    },
+    # same as the default of "spm"
```

Comment:
Why is the same model available under two names and two different signatures?

Reply:
The newer name is more descriptive, the older one for backwards compatibility. Maybe not a good idea?

Reply:
Do we need backward compatibility here? Maybe we can deprecate "spm" - switch to "flores101" (including the signature), but issue a warning that "spm" will be removed in the next version of sacrebleu.

Reply:
It's listed in § 5.2 in the [flores 101 paper](https://aclanthology.org/2022.tacl-1.30.pdf), so I'd hate to remove it entirely. But in hindsight it's definitely named too generically.

Reply:
I see there just a sentence "SpBLEU is integrated into sacrebleu for ease of use as the spm tokenizer". I don't see `-tok spm`. If you think so, we don't need to deprecate `-tok spm`, we can just keep it as an alias for `-tok flores101`. My main point is that the signature should be clear (and I think two signatures for the same thing may be confusing).

Reply:
<img width="323" alt="image" src="https://user-images.githubusercontent.com/455256/181668009-507ae792-d127-4b48-9518-4edf4703c8ec.png">

I'll give it some thought, and clean it up somehow. I agree it's simpler to have one name, and maybe we can just print a deprecation warning.

Reply:
This is what I plan to do (retain `spm` for backward compat). Sound good?

Reply:
So to be clear: You can use `-tok spm|flores101|flores200`, which will generate signatures of "flores101", "flores101", and "flores200", respectively.

Reply:
There is also a note printed if you use `-tok spm`, that the signature has changed.